Makefile

```diff
@@ -349,13 +349,21 @@ devdoc-update:
 ###                                Protobuf                                 ###
 ###############################################################################
 
+# Get clang-format binary name
+ifeq ($(UNAME_S),Linux)
+  CLANG_FORMAT_BIN ?= $(shell [ -f /etc/debian_version ] && echo "clang-format-6.0" || echo "clang-format")
+endif
+ifeq ($(UNAME_OS),Darwin)
+  CLANG_FORMAT_BIN ?= clang-format
+endif
+
```

Comment:
Please remove this. This is absolutely unnecessary and should not live here - at least not in this form.

Reply:
Just can just assume `clang-format` is a symlink to `clang-format-6.0`